src/components/OurCoreValue.tsx

```diff
@@ -0,0 +1,33 @@
+import "./OurCoreValue.css";
+
+const OurCoreValue = () => (
+  <div className="site-main-inner-wrapper-core">
+    <section>
+      <h2>Our Core Value</h2>
+      <p>
+        Are you passionate about mental health and technology? Are you looking
+        for an opportunity to gain experience and make a difference in the
+        field? We are excited to announce that Heal Health is currently seeking
+        new members to join our team on a voluntary or unpaid internship basis.
+      </p>
+      <p>
+        As a startup, we are dedicated to creating innovative and effective
+        mental health technology to support those in need. We are a small team
+        of dedicated individuals, but we are looking to grow and expand our
+        capabilities. We are looking for volunteers and interns who are eager to
+        learn and contribute their skills in areas such as product development,
+        marketing, research, sales, finances, and more.
+      </p>
+      <p>
+        We value diversity and welcome applicants with a wide range of
+        backgrounds and experiences. If you're interested in joining our team
```

Comment:
Pay attention to using the typographically correct apostrophes in text.

Instead of `'`, use `’`, it will make the text look so much nicer!